hdrs/extchat.h

```diff
@@ -84,7 +84,12 @@ struct chanuser {
 #define CB_QUIET      0x80      /* Do not prepend the <Channel> name */
 #define CB_NOCOMBINE  0x100     /* Don't send this message to players with
                                  * their channels set COMBINE */
+#define CB_SEEALL     0x200     /* Only see_all players can see this message */
 
+/* channel_buffer types */
+#define CBTYPE_SEEALL   0x01    /* Only see_all players can see this message
```

Comment:
This should just be 1, not 0x01, if it's not a bit flag. (If it is a bitflag, it should be checked with (foo | CBTYPE_SEEALL) later, not (foo == CBTYPE_SEEALL)